src/scripts/components/foundations/svgs/MainSvg.js

```diff
@@ -72,6 +72,9 @@ class MainSvg extends Component<Props, void> {
             fill="#fff"
           />
         </symbol>
+        <symbol id="icon-close" viewBox="0 0 512 512">
```

Comment:
Can we add new SVGs to `/assets/img`? I'm not sure I like the idea of building a giant SVG that will be loaded into the DOM on every render.

Reply:
This is only for reusable svgs. It doesn't need to make a request for every icon. It's pretty small, in fact

Reply:
Gotcha. Maybe in the future it could grow very large but fine for now. 

My only gripe now is that we are loading/managing SVGs in two different ways now, so we’ll want to reconcile that.

Reply:
The MainSvg holds the icons basically. The others svgs are set directly in the html.